07_web_endpoints/discord_bot.py

```diff
@@ -0,0 +1,230 @@
+# ---
+# lambda-test: false
+# ---
+
+# # Serve a Discord Bot on Modal
+
+# (quick links: [try it out on Discord](https://discord.gg/nR96BxPu))
```

Comment:
I think it's a good idea to host an example, but we should take care to:
1. Make it a Modal-branded Discord server.
2. Restrict all permissions for members except running slash commands.